### PR TITLE
Fixes celery and upload dbf

### PR DESCRIPTION
**Pull request by esloch**
- Updated _datetime to_pydatetime() and pd.tslib.nattype to pd.NaT in the sinan.py file
- Modified dbf database connection for infodengue in the settings.py file
- Changed os.getenv () method in settings.py EMAIL_CONFIG, inserted .split(' ') Method
- Pinned django-chunked-upload=1.1.3
- Optimized docker-celery image (Reduced celery image layer size)
- Inserted into script 'CREATE ROLE Read_only' in infodengue_roles

### CODE REVIEW

File: AlertaDengue/AlertaDengue/__init__.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
-# coding=utf-8
-from __future__ import absolute_import, unicode_literals
+# -*- coding: utf-8 -*-
```
Comment (fccoelho) [inline AlertaDengue/AlertaDengue/__init__.py]: Isto  é coisa de Python2, pode remover.

File: AlertaDengue/dbf/sinan.py [hunk 2]
```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```
Comment (fccoelho) [inline AlertaDengue/dbf/sinan.py]: idem. Remover todos estes comentários de `coding`

File: AlertaDengue/AlertaDengue/__init__.py (hunk 1 above)
Comment (esloch) [reply AlertaDengue/AlertaDengue/__init__.py]: ok.

File: AlertaDengue/AlertaDengue/settings.py [hunk 3]
```diff
@@ -130,6 +130,7 @@ def read_admins(value):
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
         'NAME': PSQL_DBF,
+        'NAME': PSQL_DB,
```
Comment (fccoelho) [inline AlertaDengue/AlertaDengue/settings.py]: decidir qual o valor correto de `NAME`

Comment (esloch) [reply AlertaDengue/AlertaDengue/settings.py]: Ficará como PSQL_DBF, se refere a base infodengue onde registra os metadados de dbf.

**Pull request merged**